src/app.module.ts

```diff
@@ -21,6 +26,7 @@ import { CheckEmailService } from './services/check-email.service';
 })
 export class AppModule {
   configure(consumer: MiddlewareConsumer) {
+    consumer.apply(APMAgent.init());
```

Comment:
should connect the middleware like this
`apm.agent.middleware.connect()`

Reply:
Ok i see .thanks https://www.elastic.co/guide/en/apm/agent/nodejs/master/agent-api.html#apm-middleware-connect